src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -350,11 +353,11 @@ public void Socket_SendToAsyncV4IPEndPointToV4Host_Throws()
         {
             Socket socket = new Socket(AddressFamily.InterNetworkV6, SocketType.Dgram, ProtocolType.Udp);
             SocketAsyncEventArgs args = new SocketAsyncEventArgs();
-            args.RemoteEndPoint = new IPEndPoint(IPAddress.Loopback, TestPortBase + 25);
+            args.RemoteEndPoint = new IPEndPoint(IPAddress.Loopback, TestPortBase + 40);
             args.SetBuffer(new byte[1], 0, 1);
             bool async = socket.SendToAsync(args);
             Assert.False(async);
-            Assert.Equal(SocketError.Fault, args.SocketError);
+			Assert.NotEqual(SocketError.Success, args.SocketError);
```

Comment:
I'll revert this to its original form and mark this with an issue. We'll need to do the work to return the same error here if possible.